syftlib/src/main/java/org/openmined/syft/Syft.kt

```diff
@@ -19,6 +19,15 @@ import java.util.concurrent.atomic.AtomicBoolean
 
 private const val TAG = "Syft"
 
+/**
+ * The Syft Class is the main worker handling all the Federated Training Cycles. 
```

Comment:
- main worker class
- Only one instance of the worker should be initialized to manage creation and deletion of one or many jobs. We recommend android devices to not create multiple jobs as usually they are resource-intensive and may hamper user experience.